tests/array.js

```diff
@@ -3,6 +3,12 @@ var expect = chai.expect;
 var jsonexport = require('../lib/index');
 
 describe('Array', () => {
+    it('options omitted', () => {
```

Comment:
@kauegimenes this is the only location that actually catches the original bug (once options is mistakenly set at the global level, it will pass)

I'll add a use strict annotation to enforce it instead?